.github/workflows/e2e-embed.yml

```diff
@@ -74,6 +74,7 @@ jobs:
           SENDGRID_EMAIL: ${{ secrets.CI_SENDGRID_EMAIL }}
           TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
           TURBO_TEAM: ${{ secrets.TURBO_TEAM }}
+          DAILY_API_KEY: ${{ secrets.CI_DAILY_API_KEY }}
```

Comment:
These were put in alphabetical order on our oírse in all of these files. Let’s maintain that